### ISSUE TITLE
README.md - add command to the 'Resetting the app' paragraph

### ISSUE THREAD

**Issue by j-ed** [nextcloud/mail#523]
In the README.md file the paragraph "**Resetting the app**" exists which describes which commands need to be executed to cleanse all database tables. Unfortunately the table `oc_mail_attachments` is not 
listed. To really remove "**all**" tables the command `DROP TABLE oc_mail_attachments;` need to be added.
```
DELETE FROM oc_appconfig WHERE appid = 'mail';
DROP TABLE oc_mail_accounts;
DROP TABLE oc_mail_aliases;
DROP TABLE oc_mail_collected_addresses;
DROP TABLE oc_mail_attachments;                 # <<<
```

**Comment by ChristophWurst** [comment]
@j-ed yes, right, that table's missing. Mind to create a PR and fix it? GitHub even allows you to do so via the web UI 😉

**Comment by j-ed** [comment]
*WOW* easier than I thought. See https://github.com/nextcloud/mail/pull/524

**Comment by ChristophWurst** [comment]
Fixed in #524 😉

**Comment by ChristophWurst** [comment]
From bug report to merged PR in under 15min. We've set a new record 💃
